docs/UserGuide.md

```diff
@@ -25,7 +25,7 @@ Assistant who can type fast, TAilor can get your contact management tasks done f
 
 4. Double-click the file to start the app. The GUI similar to the below should appear in a few seconds. Note
    how the app contains some sample data.<br>
-   ![Ui](images/Ui.png)
+   ![Ui](images/GUI.png)
```

Comment:
This should still be named as Ui.png though, its part of one of the requirements i think, we can't rename this

Reply:
this change was made for our 2101 UGDG review yesterday so I can revert it back to our mockup png now:)

Reply:
Hi just to update, I have renamed GUI.png to Ui.png in the recent commits since we require a realistic image at this stage of documentation. Happy to replace it with another screenshot (without dummy information) later this week:)